lib/stack_master/parameter_resolvers/one_password.rb

```diff
@@ -17,12 +17,25 @@ def resolve(params={})
 
       private
 
+      def validate_op_installed?
+        %x(op --version)
+        rescue Errno::ENOENT => exception
+          raise RuntimeError, "The op cli needs to be installed and in the PATH, #{exception}"
```

Comment:
Should these two lines be indented one level less? Ruby isn't white space sensitive, so it doesn't matter but for style ya know